src/session/index.ts

```diff
@@ -311,7 +311,7 @@ export default abstract class Session extends EventEmitter {
         }
     }
 
-    setMock (requestFilterRule: RequestFilterRule, mock: ResponseMock): void {
+    async setMock (requestFilterRule: RequestFilterRule, mock: ResponseMock): Promise<void> {
```

Comment:
Shouldn't we reflect this change in the typings ([#1](https://github.com/DevExpress/testcafe-hammerhead/blob/1e394735c7a1275feb767817e8df0eeee435b4b2/ts-defs/index.d.ts#L85), [#2](https://github.com/DevExpress/testcafe-hammerhead/blob/1e394735c7a1275feb767817e8df0eeee435b4b2/ts-defs/index.d.ts#L187)) as well?